cdap-api/src/main/java/co/cask/cdap/api/data/schema/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Defines schema for use in programs and data exploration.
+ */
+package co.cask.cdap.api.data.schema;
```

Comment:
Are you planning to move this https://github.com/caskdata/cdap/blob/develop/cdap-api/src/main/java/co/cask/cdap/internal/io/package-info.java into here as well?